types/ghauth/ghauth-tests.ts

```diff
@@ -0,0 +1,27 @@
+import ghauth, { AuthOptions, TokenData } from "ghauth";
```

Comment:
Default ES6 imports aren't callable.  Seems like you might have wanted
```ts
import ghauth = require("ghauth");
import { AuthOptions, TokenData } from "ghauth";
```

Reply:
Thank you for your review.
Done.

Reply:
Thanks!